docs/user_guide.md

```diff
@@ -119,54 +119,65 @@ You can check one or more members, then click `ACTION`, choose one role to batch
 
-#### Trigger mode
-* **Manual**: Replicate the repositories manually when needed. **Note**: The deletion operations are not replicated. 
-* **Immediate**: When a new repository is pushed to the project, it is replicated to the remote registry immediately. Same to the deletion operation if the `Delete remote images when locally deleted` checkbox is selected.
-* **Scheduled**: Replicate the repositories daily or weekly. **Note**: The deletion operations are not replicated. 
-
-![browse project](img/create_rule.png)
+**Note:** `library` must be added if you want to replicate the official images of Docker Hub. For example, `library/hello-world` matches the official hello-world images.  
 
-### Listing and stopping replication jobs
-Click a rule, jobs which belong to this rule will be listed. A job represents the progress of replicating the repository to the remote instance. Click `STOP JOBS`, the pending and retrying jobs will be stopped immediately and the running jobs will be canceled at the next checkpoint.  
+Pattern | String(Match or not)
+---------- | -------
+`library/*`      | `library/hello-world`(Y)<br> `library/my/hello-world`(N)
+`library/**`     | `library/hello-world`(Y)<br> `library/my/hello-world`(Y)
+`{library,goharbor}/**` | `library/hello-world`(Y)<br> `goharbor/harbor-core`(Y)<br> `google/hello-world`(N)
+`1.?`      | `1.0`(Y)<br> `1.01`(N)
 
-![browse project](img/list_stop_jobs.png)
+#### Trigger mode
+* **Manual**: Replicate the resources manually when needed. **Note**: The deletion operations are not replicated. 
+* **Scheduled**: Replicate the resources periodically. **Note**: The deletion operations are not replicated. 
+* **Event Based**: When a new resource is pushed to the project, it is replicated to the remote registry immediately. Same to the deletion operation if the `Delete remote resources when locally deleted` checkbox is selected.
 
 ### Starting a replication manually
-Select a replication rule and click `REPLICATE`, the images under the project which the rule is applied to will be replicated to the remote registry immediately. If there is any pending/running job that belongs to the rule, the new replication will not be started.
+Select a replication rule and click `REPLICATE`, the resources which the rule is applied to will be replicated from the source registry to the destination immediately.  
 
 ![browse project](img/start_replicate.png)
 
-### Deleting the replication rule
-Select the replication rule and click `DELETE` to delete it. Only rules which have no pending/running/retrying jobs can be deleted.  
+### Listing and stopping replication executions
+Click a rule, the execution records which belong to this rule will be listed. Each record represents the summary of the once execution of the rule. Click `STOP` to stop the executions which are in progress.  
 
-![browse project](img/delete_rule.png)
+![browse project](img/list_stop_executions.png)
 
+### Listing tasks

```

Comment:
Suggest adding some text to describe the status meanings, especially `In Progress` because it includes the 'Pending' status.